### ISSUE TITLE
Limit "allowlisted" rule to "only allow specific request url"

### ISSUE THREAD

**Issue by WZ-Tong** [AdguardTeam/AdguardFilters#171993]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard Browser Extension, AdGuard Content Blocker, AdGuard for Windows

### AdGuard version

7.16.0 (4542) (CL 1.13.115, DL 2.4.37 )

### What type of problem have you encountered?

Missed ads or ad leftovers

### Which browser(s) do you use?

Chrome, Microsoft Edge

### Which device do you use?

Desktop

### Where is the problem encountered?

bing to https://whatmyuseragent.com/

### Ad Blocking

AdGuard Base filter, EasyList

### Privacy

AdGuard URL Tracking filter, AdGuard Tracking Protection filter, EasyPrivacy

### What Stealth Mode options do you have enabled?

Hide your search queries, Send Do-Not-Track signals, Self-destruction of third-party cookies, Self-destruction of first-party cookies, Disable cache for third-party requests, Block third-party Authorization header, Block WebRTC, Block Push API, Block Location API, Hide Referrer from third-parties (specify in comment below, if used custom value), Hide your User-Agent (specify in comment below, if used custom value), Mask your IP address (specify in comment below, if used custom value), Remove X-Client-Data header from HTTP requests, Protect from DPI

### Add your comment and screenshots

1. Comment
When I directly access target, it triggers rules,
When I access via bing, it triggers bing's whitelist.
I think by using `@@||example.com`, it only unblocks when `request url` match, rather than other fields (in this case, `source url`)

2. Screenshots
<details><summary>Screenshot 1:</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/77199564/5ab183c3-c6f1-4edd-a34b-2f4a7b270bb5)

</details>

<details><summary>Screenshot 2:</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/77199564/50c1d941-3d62-45b5-ae48-9ff3ac4204f3)

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by Alex-302** [comment]
Please describe step by step.
What I see when search whatmyuseragent.com an go to the site with the rule `@@||bing.com^$important,document`

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/513b00a4-9718-4754-ae15-153213bc6443)

**Comment by WZ-Tong** [comment]
You're right, now I noticed that this only happens when I use `Tun Mode` in Clash Verge: [GitHub Repo](https://github.com/clash-verge-rev/clash-verge-rev) or [Clash.Verge_1.4.10_x64-setup.exe](https://github.com/clash-verge-rev/clash-verge-rev/releases/download/v1.4.10/Clash.Verge_1.4.10_x64-setup.exe)

**Step to reproduce:**
1. Install Clash Verge (only 1.4.10 tested)
2. `Settings` -> `System Settings` -> Turn on `Tun mode` and `Service mode`, my setting is listed below:
![Clash Verge Setting](https://github.com/AdguardTeam/AdguardFilters/assets/77199564/b73bebe1-818f-44d0-bd56-46859720906e)
3.  `Profile` -> `Profile URL` -> My private link sent to [alex302@adguard.com](mailto:alex302@adguard.com) with email [wztong@vip.qq.com](mailto:wztong@vip.qq.com) -> `Download`
4. `Proxies` -> `Global`
5. `Proxies` -> `Delay check` -> Pick a valid node below
![image](https://github.com/AdguardTeam/AdguardFilters/assets/77199564/7bcf5952-f767-4147-b660-a1cf85339fe0)
You may find the tray icon turns green like:
![image](https://github.com/AdguardTeam/AdguardFilters/assets/77199564/f7039b6d-00ac-46dc-a818-a15496758f0f)
6. Seach `user agent` in bing, jump to `https://whatmyuseragent.com/`, maybe with `Stealth mode: Hide my UA` enabled.
7. The website display the **original** UA of edge.
8. Open a new tab and enter `https://whatmyuseragent.com/`.
9. The website display the **modified** UA of edge.

**Note:**
1. The VPN service I use (the subscribe link in email) is provided by **third party**, the traffic through it **may be logged** by the provider, please disconnect any sensitive connections/softwares before connecting/opening Clash Verge.
2. After using it, close the service by right clicking tray icon rather than directly close it.
